languages/python/oso/tests/test_polar.py

```diff
@@ -835,3 +836,23 @@ class Post:
 
     with pytest.raises(StopIteration):
         next(results)
+
+
+def test_iterators(polar, qeval, qvar):
+    assert qvar('x in "abc"', "x") == ["a", "b", "c"]
+    assert qvar("x in {a: 1, b: 2}", "x") == ["a", "b"]
```

Comment:
It's unfortunate that these features are totally language-specific, even though they appear to be builtin.